src/parser.pest

```diff
@@ -6,9 +6,10 @@ comment     = @{ "`"  ~ ("\\" ~ ANY | !"`" ~ ANY)*  ~ "`" }
 empty       = @{ "!" }
 
 sequence    =  { "[" ~ list ~ "]" }
-stack       =  { "{" ~ list ~ "}" }
-choice      =  { "<" ~ list ~ ">" }
-list        = _{ lbox_expr ~ ("," ~ lbox_expr)* }
+stack       =  { "{" ~ comma_list ~ "}" }
+choice      =  { "<" ~ comma_list ~ ">" }
+list        = _{ lbox_expr ~ (","* ~ lbox_expr)* }
```

Comment:
Why make it optional and not remove it at all? Also, why only in `list`, which ends up in `sequence` but not in `stack` and choice? Doesn't the current syntax `","` allow for `lbox_expr,,,,lbox_expr` ?